poulpe-model/src/test/java/org/jtalks/poulpe/model/dao/hibernate/UserHibernateDaoTest.java

```diff
@@ -50,7 +55,9 @@
 @TransactionConfiguration(transactionManager = "transactionManager", defaultRollback = true)
 @Transactional
 public class UserHibernateDaoTest extends AbstractTransactionalTestNGSpringContextTests {
-	static final String NO_FILTER = "";
+    static final String NO_FILTER = "";
+    private final String USERNAME = between(5,20).with(prefix("a")).english(); // prefix is used to have different capitalized username
```

Comment:
No need to make these global - they can be local to the method. That way it's easier to see what's happening in each test.